identifiers/country-in/lok_sabha.csv

```diff
@@ -1,546 +1,548 @@
+ocd-division/country:in/state:wb/cd:diamond_harbour,West Bengal Lok Sabha constituency Diamond Harbour,,
+ocd-division/country:in/state:wb/cd:dum_dum,West Bengal Lok Sabha constituency Dum Dum,,
+ocd-division/country:in/state:wb/cd:ghatal,West Bengal Lok Sabha constituency Ghatal,,
+ocd-division/country:in/state:wb/cd:hooghly,West Bengal Lok Sabha constituency Hooghly,,
+ocd-division/country:in/state:wb/cd:howrah,West Bengal Lok Sabha constituency Howrah,,
+ocd-division/country:in/state:wb/cd:jadavpur,West Bengal Lok Sabha constituency Jadavpur,,
+ocd-division/country:in/state:wb/cd:jalpaiguri,West Bengal Lok Sabha constituency Jalpaiguri (SC),,
+ocd-division/country:in/state:wb/cd:jangipur,West Bengal Lok Sabha constituency Jangipur,,
+ocd-division/country:in/state:wb/cd:jaynagar,West Bengal Lok Sabha constituency Jaynagar (SC),,
+ocd-division/country:in/state:wb/cd:jhargram,West Bengal Lok Sabha constituency Jhargram (ST),,
+ocd-division/country:in/state:wb/cd:kanthi,West Bengal Lok Sabha constituency Kanthi,,
+ocd-division/country:in/state:wb/cd:kolkata_dakshin,West Bengal Lok Sabha constituency Kolkata Dakshin,,
+ocd-division/country:in/state:wb/cd:kolkata_uttar,West Bengal Lok Sabha constituency Kolkata Uttar,,
+ocd-division/country:in/state:wb/cd:krishnanagar,West Bengal Lok Sabha constituency Krishnanagar,,
+ocd-division/country:in/state:wb/cd:maldaha_dakshin,West Bengal Lok Sabha constituency Maldaha Dakshin,,
+ocd-division/country:in/state:wb/cd:maldaha_uttar,West Bengal Lok Sabha constituency Maldaha Uttar,,
+ocd-division/country:in/state:wb/cd:mathurapur,West Bengal Lok Sabha constituency Mathurapur (SC),,
+ocd-division/country:in/state:wb/cd:medinipur,West Bengal Lok Sabha constituency Medinipur,,
+ocd-division/country:in/state:wb/cd:murshidabad,West Bengal Lok Sabha constituency Murshidabad,,
+ocd-division/country:in/state:wb/cd:purulia,West Bengal Lok Sabha constituency Purulia,,
+ocd-division/country:in/state:wb/cd:raiganj,West Bengal Lok Sabha constituency Raiganj,,
+ocd-division/country:in/state:wb/cd:ranaghat,West Bengal Lok Sabha constituency Ranaghat (SC),,
+ocd-division/country:in/state:wb/cd:srerampur,West Bengal Lok Sabha constituency Srerampur,,
+ocd-division/country:in/state:wb/cd:tamluk,West Bengal Lok Sabha constituency Tamluk,,
+ocd-division/country:in/state:wb/cd:uluberia,West Bengal Lok Sabha constituency Uluberia,,
+ocd-division/country:in/territory:an/cd:andaman_and_nicobar_islands,Andaman and Nicobar Islands Lok Sabha constituency Andaman and Nicobar Islands,,
+ocd-division/country:in/territory:ch/cd:chandigarh,Chandigarh Lok Sabha constituency Chandigarh,,
+ocd-division/country:in/territory:dd/cd:daman_and_diu,Daman and Diu Lok Sabha constituency Daman and Diu,,2020-01-25
+ocd-division/country:in/territory:dh/cd:dadra_and_nagar_haveli,Dadra and Nagar Haveli and Daman and Diu Lok Sabha constituency Nagar Haveli,2020-01-26,

```

Comment:
I'm wondering whether the constituencies in the new union territory should be consisdered new districts or if they should be considered new canonical names for the old districts?

Are there other examples where a new union territory was created and the constituencies remained unchanged? If there are, were officeholders considered to have a continuous tenure representing the same district for those involved?